test/ssl_test.c

```diff
@@ -233,7 +233,9 @@ static int execute_test(SSL_TEST_FIXTURE fixture)
         client_ctx = SSL_CTX_new(DTLS_client_method());
         if (test_ctx->handshake_mode == SSL_TEST_HANDSHAKE_RESUME) {
             resume_server_ctx = SSL_CTX_new(DTLS_server_method());
-            OPENSSL_assert(resume_server_ctx != NULL);
+            resume_client_ctx = SSL_CTX_new(DTLS_client_method());
+            OPENSSL_assert(resume_server_ctx != NULL
```

Comment:
I’d be leary of using all-caps ASSERT in the name, since it might confuse readers who think the code is getting compiled-out.  Just a side-comment.